From a3c117bf06c78c8eca4b4cedcda769a0791f2f08 Mon Sep 17 00:00:00 2001
From: Andrew Boie <andrew.p.boie@intel.com>
Date: Mon, 6 Aug 2012 10:49:08 -0700
Subject: init: remove logo code

This code is legacy and should never need to be used. The bootloader
should put an image on the screen and the kernel shouldn't reconfigure
the display controller until userland is ready to draw something,
such as the boot animation, charger, recovery console, etc.

Instead of hard-coding tty0, the "A N D R O I D" text is written out
to whatever the active console is.

Change-Id: I5fd21413f721d2ba6966d34d632d8fc3913ca073
Signed-off-by: Andrew Boie <andrew.p.boie@intel.com>
---
 init/Android.mk |    1 -
 init/init.c     |    2 --
 init/init.h     |    4 ----
 3 files changed, 0 insertions(+), 7 deletions(-)

diff --git a/init/Android.mk b/init/Android.mk
index 67a9c8d..53d3524 100644
--- a/init/Android.mk
+++ b/init/Android.mk
@@ -10,7 +10,6 @@ LOCAL_SRC_FILES:= \
 	property_service.c \
 	util.c \
 	parser.c \
-	logo.c \
 	keychords.c \
 	signal_handler.c \
 	init_parser.c \
diff --git a/init/init.c b/init/init.c
index 92ed88d..4eeec8e 100755
--- a/init/init.c
+++ b/init/init.c
@@ -573,8 +573,6 @@ static int console_init_action(int nargs, char **args)
         console_name = strdup(tmp);
     }
 
-    load_565rle_image(INIT_IMAGE_FILE);
-
     /* Block the boot until the console node comes up */
     while (1) {
         fd = open(console_name, O_WRONLY);
diff --git a/init/init.h b/init/init.h
index 955e1f0..2f1ab2b 100644
--- a/init/init.h
+++ b/init/init.h
@@ -130,10 +130,6 @@ void service_reset(struct service *svc);
 void service_start(struct service *svc, const char *dynamic_args);
 void property_changed(const char *name, const char *value);
 
-#define INIT_IMAGE_FILE	"/initlogo.rle"
-
-int load_565rle_image( char *file_name );
-
 extern struct selabel_handle *sehandle;
 extern struct selabel_handle *sehandle_prop;
 extern int selinux_reload_policy(void);
-- 
1.7.4.1

